### DISCUSSION TITLE
How to prevent custom styles from being parsed into CSS

### DISCUSSION THREAD

**Discussion by momu-2016** [GrapesJS/grapesjs#4989] [Q&A]
I added a custom property in StyleManager for some special reason, but I don't want this property to be rendered as CSS in the end. Is there any solution for this? this is my code,Thanks.
` const sm = editor.StyleManager;
sm.getSectors().reset([
 name: "Position",
                open: false,
                properties: [
                    {name: "Relative to",
                        requires: { position: ["relative", "absolute", "fixed", "sticky"] },
                        property: "relative-to",
                        defaults: "itself",
                        type: 'display-label',
                        ignore:true,
                        title: "The container you're positioning the selected item relative to."
                    },
])`

**Comment by artf** [comment]
There is actually one trick to avoid the property being rendered in CSS, you should prefix the name of the property with `__`, 
eg. `property: '__relative-to'`
